lib/crypto/base32.js

```diff
@@ -21,7 +21,7 @@ function base32(len) {
     .then(bytes => {
       const out = []
 
-      for (let i = 0; i < len; i++) {
+      for (let i = 0; i < len; i ++) {
```

Comment:
@divyabiyani so let's reopen a new PR, we can also include this rule in the new PR: https://github.com/pdehaan/eslint-config-fxa/pull/11/files